docs/getting_started.md

```diff
@@ -29,42 +30,47 @@ Once you've created your new repository, the creation process will tell you to:
 2. Configure upstream / remote repository:
 
     a. if you have an existing repository:
+
     ```bash
     git remote add origin https://github.com/{{organization}}/{{project_name}}.git
     git pull origin main  # and resolve any conflicts
     ```
+
     To resolve conflicts, you can rename conflicting files (like README.md) and then solve
     conflicts, or, continue without pulling, commit your changes, and then pull with
     `git pull origin main --allow-unrelated-histories` and solve the conflicts then.
 
     b. or, create a new repository (install [GitHub CLI](https://cli.github.com/) if necessary)
+
     ```bash
     gh auth login
     gh repo create {{organization}}/{{project_name}} --private --source=. --remote=origin
     ```
 
 3. Install dependencies:
+
     ```bash
-    pdm add <dependencies>
-    pdm install
+    uv add <dependencies>
     ```
 
 4. Commit the initial files:
+
     ```bash
     git add .
-    pre-commit install --install-hooks  # only if you chose to get pre-commit and have it installed
+    uvx --with pre-commit-uv -- pre-commit install --install-hooks  # only if you chose to get pre-commit and have it installed
```

Comment:
same comment as above regarding installing rather than using uvx (for consistent pre-commit version across projects)